interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1553,7 +1554,7 @@ def get_yaml(self, out):
         pass
 
 
-def convert(filename=None, output_name=None, text=None):
+def convert(filename=None, output_name=None, text=None, header=True):
```

Comment:
I don't think that this needs to be configurable. If a user doesn't want the description in the output, they can remove it from the output file manually.

Reply:
Removed.